content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -142,9 +131,7 @@ If you set streaming up with [CloudFormation](?tab=cloudformation#installation):
 
 If you set streaming up through the [AWS Console](?tab=awsconsole#installation):
 1. Delete the CloudWatch Metric Stream linked to your delivery stream.
-2. Delete the Kinesis Data Firehose delivery stream that delivered metrics to Datadog.
-3. Delete the backup S3 bucket for failed messages linked to the Firehose.
-4. Delete the IAM roles associated with the stream and all other resources that were created while setting up the stream.
+2. Delete the S3 Bucket, Firehose, IAM roles associated with the stream and all other resources that were created while setting up the stream.
```

Comment:
Just to confirm, this isn't saying to also delete "all other resources that were created while setting up the stream", right? That's how I first read the sentence but now I'm not sure if that's correct. Are we only talking about deleting roles here?

Reply:
You got it right - it does say to delete all resources that were created while setting up the stream! Thanks!